src/views/blog/admin/path_list.vue

```diff
@@ -165,86 +361,273 @@ export default class PathList extends Vue {
     this.paths = {};
     AdminAuth.attempt_get_JWT()
       .then((token) => {
-        api(this.client)
-          .blog.articles.$get()
-          .then((data: BlogArticle[]) => {
-            for (const article of data) {
-              this.$set(this.paths, article.id, {
-                category: article.category,
-                title: article.title,
-                created_at: article.created_at,
-                updated_at: article.updated_at,
-                waiting_count: 0,
-              });
-            }
-          });
-        return token;
+        // get articles
+        return new Promise<JWT>((resolve) =>
+          api(this.client)
+            .blog.articles.$get()
+            .then((data: BlogArticle[]) => {
+              for (const article of data) {
+                this.$set(this.paths, article.id, {
+                  category: article.category,
+                  title: article.title,
+                  author: article.author,
+                  handle_name: article.handle_name,
+                  created_at: article.created_at,
+                  updated_at: article.updated_at,
+                  waiting_count: 0,
+                  revision_id: article.revision_id,
+                  article_exists: true,
+                  revisions: [],
+                  revision_selection: article.revision_id,
+                  category_selection: article.category,
+                });
+              }
+              resolve(token);
+            })
+        );
       })
       .then((token) => {
+        // get revisions
         api(this.client)
           .blog.revisions.$get({
             headers: {
               "X-ADMIN-TOKEN": token.content,
             },
           })
-          .then((data: BlogRevision[]) => {
+          .then((data: TableRevision[]) => {
             for (const revision of data) {
               if (!(revision.article_id in this.paths)) {
                 // does not exist
                 this.$set(this.paths, revision.article_id, {
                   title: revision.title,
+                  created_at: revision.timestamp,
+                  author: revision.author,
+                  handle_name: revision.handle_name,
+                  article_exists: false,
                   waiting_count: 0,
+                  revision_id: revision.id,
+                  revisions: [],
```

Comment:
idが余計になっちゃうのは問題なさそう(想像) 型で怒られるかなぁ?
```suggestion
                  ...revision,
                  article_exists: false,
                  waiting_count: 0,
                  revision_id: revision.id,
                  revisions: [],
```